packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -524,7 +524,9 @@ books.addMethod('GET', new apigateway.HttpIntegration('http://amazon.com'), {
 });
 ```
 
-You can find a full working example [here](test/authorizers/integ.token-authorizer.ts).
+A full working example is shown below.
+
+[Full token authorizer example](test/authorizers/integ.token-authorizer.lit.ts).
```

Comment:
You will also need to change the name of the two files to have the extension `.lit.ts`.